src/Compilers/VisualBasic/Portable/Operations/VisualBasicArgument.vb

```diff
@@ -0,0 +1,62 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Semantics
+
+Namespace Microsoft.CodeAnalysis.VisualBasic
+    Friend MustInherit Class BaseVisualBasicArgument
+        Inherits BaseArgument
+
+        Protected Sub New(argumentKind As ArgumentKind, parameter As IParameterSymbol, inConversion As Conversion, outConversion As Conversion, semanticModel As SemanticModel, syntax As SyntaxNode, type As ITypeSymbol, constantValue As [Optional](Of Object), isImplicit As Boolean)
+            MyBase.New(argumentKind, parameter, semanticModel, syntax, type, constantValue, isImplicit)
+
+            InConversionInternal = inConversion
+            OutConversionInternal = outConversion
+        End Sub
+
+        Friend ReadOnly Property InConversionInternal As Conversion
+
+        Friend ReadOnly Property OutConversionInternal As Conversion
+
+        Public Overrides ReadOnly Property InConversion As CommonConversion
+            Get
+                Return If(InConversionInternal.Exists, InConversionInternal.ToCommonConversion(), New CommonConversion(exists:=True, isIdentity:=True, isNumeric:=False, isReference:=False, methodSymbol:=Nothing))
```

Comment:
>If(InConversionInternal.Exists, InConversionInternal.ToCommonConversion(), New CommonConversion(exists:=True, isIdentity:=True, isNumeric:=False, isReference:=False, methodSymbol:=Nothing)) [](start = 23, length = 189)

This logic feels strange. Do we ever get  conversion that doesn't exist? We probably should return it anyway.